### DISCUSSION TITLE
Walking + Bike route

### DISCUSSION THREAD

**Discussion by Garate77** [organicmaps/organicmaps#2500] [Q&A]
We are trying to implement a functionality that identifies in our city (Madrid, Spain) how to ride a bike, knowing that in some parts of the route the user will have to walk. Is it possible to do it and do you know where can we look into to display different colors for each leg of the route?

**Comment by biodranik** [comment]
Sorry, it is not clear: do you plan to use OM for it, or asking about some other app/code?

**Reply by Garate77** [reply]
We are passing a list of checkpoints to Organic Maps using an API, indicating which ones are to be done walking or riding, so the resulting route has to be hibrid, similar to what transit does.

**Reply by biodranik** [reply]
It is not supported yet, but it's a good idea. Can you please create an issue with the detailed description/requirements? We have plans to fix/update the API.
